tests/test_batch.py

```diff
@@ -55,7 +57,7 @@ def test_batch_mode_01(self):
         #for s in outstr:
         #    print(s)
         self.maxDiff = None
-        assert outstr == []
+        #DIT WAS NIET ZO, BESPREKEN MET BERT! assert outstr == []
```

Comment:
is dit besproken met Bert? 
- zo ja wat is de uitkomst?
- zo nee, bespreek dit met bert.

Reply:
ok... no clue but all works fine now

Reply:
removed comment and enabled assertion